src/OnboardingSPA/components/Drawer/DrawerPanel/index.js

```diff
@@ -35,6 +35,7 @@ import { store as nfdOnboardingStore } from '../../../store';
 import WithDesignBack from './WithDesignBack';
 
 const DrawerPanel = () => {
+	// eslint-disable-next-line no-unused-vars
```

Comment:
Why are we still using this state? Please check and remove it if possible, I don't see any issue.